engineyard/ey-cookbooks-dev-v7
# Updated paths for elasticsearch Heapdumps and logs

Description of your patch
-------------
The patch mounts heapdumps default directory to `/data/elasticsearch` and elasticsearch logs to `/mnt/log/elasticsearch` . This enables us to expand the volumes in case free space runs out due to elasticsearch.

Recommended Release Notes
-------------

The patch mounts heapdumps default directory to `/data/elasticsearch` and elasticsearch logs to `/mnt/log/elasticsearch`

Estimated risk
-------------

Normal

Components involved
-------------
`ey-elasticsearch`

Dependencies
-------------
None

Description of testing done
-------------
1. Boot an environment with an elasticsearch util instance
2. Overlay the `ey-elasticsearch` cookbook.
3. Re-apply recipes
4. Check where elasticsearch logs appear (should be `/mnt/log/elasticsearch`)
5. Check where elasticsearch heapdumps appear after a while (should be `/data/elasticsearch`)

QA Instructions
-------------
1. Boot an environment with an elasticsearch util instance
2. Overlay the `ey-elasticsearch` cookbook.
3. Re-apply recipes
4. Check where elasticsearch logs appear (should be `/mnt/log/elasticsearch`)
5. Check where elasticsearch heapdumps appear after a while (should be `/data/elasticsearch`)
